### PR TITLE
[HearThisAtIE] Fix extractor.

### PR DESCRIPTION
**Pull request by Ashish0804**
---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/ytdl-org/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/ytdl-org/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Bug fix
- [ ] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Closes https://github.com/yt-dlp/yt-dlp/issues/740

### CODE REVIEW

File: yt_dlp/extractor/hearthisat.py [hunk 1]
```diff
@@ -78,27 +78,23 @@ def _real_extract(self, url):
...
-            meta_span % 'comment_count', webpage, 'comment count', fatal=False))
-        duration = str_to_int(self._search_regex(
-            r'data-length="(\d+)', webpage, 'duration', fatal=False))
-        timestamp = str_to_int(self._search_regex(
-            r'<span[^>]+class="calctime"[^>]+data-time="(\d+)', webpage, 'timestamp', fatal=False))
+        view_count = str_to_int(re.findall(meta_span % 'plays_count', webpage)[-1])
+        like_count = str_to_int(re.findall(meta_span % 'likes_count', webpage)[-1])
+        comment_count = str_to_int(re.findall(meta_span % 'comment_count', webpage)[-1])
+        duration = str_to_int(re.findall(r'data-length="(\d+)', webpage)[-1])
+        timestamp = str_to_int(re.findall(r'<span[^>]+class="calctime"[^>]+data-time="(\d+)', webpage)[-1])
```
Comment (pukkandan) [inline yt_dlp/extractor/hearthisat.py:85]: this effectively makes all these fatal

Comment (Ashish0804) [reply yt_dlp/extractor/hearthisat.py:85]: bcoz of https://github.com/yt-dlp/yt-dlp/issues/740#issuecomment-902851207
I'm marking this PR as draft

**Comment by Ashish0804** [comment]
Thanks to @marburger93
Extractor is not api based.

File: yt_dlp/extractor/hearthisat.py [hunk 2]
```diff
@@ -27,13 +23,12 @@ class HearThisAtIE(InfoExtractor):
...
+            'description': 'md5:1adb0667b01499f9d27e97ddfd53852a',
             'upload_date': '20150118',
-            'comment_count': int,
-            'view_count': int,
-            'like_count': int,
+            'playback_count': int,
+            'favoritings_count': int,
             'duration': 71,
-            'categories': ['Experimental'],
+            'categories': 'Experimental',
```
Comment (pukkandan) [inline yt_dlp/extractor/hearthisat.py:31]: must be a list

File: yt_dlp/extractor/hearthisat.py [hunk 3]
```diff
@@ -27,13 +23,12 @@ class HearThisAtIE(InfoExtractor):
...
             'thumbnail': r're:^https?://.*\.jpg$',
             'timestamp': 1421564134,
-            'description': 'md5:a4b6190da9075c89b203ac369f24d66e',
+            'description': 'md5:1adb0667b01499f9d27e97ddfd53852a',
             'upload_date': '20150118',
-            'comment_count': int,
-            'view_count': int,
-            'like_count': int,
+            'playback_count': int,
+            'favoritings_count': int,
```
Comment (pukkandan) [inline yt_dlp/extractor/hearthisat.py:29]: see correct field names in common.py

Comment (Ashish0804) [reply yt_dlp/extractor/hearthisat.py:29]: imo it's better to remove these and `categories` should be changed to `genre`.

View_count and playback count are diff
same with like_count and fav count

Comment (Ashish0804) [reply yt_dlp/extractor/hearthisat.py:29]: they are not in the api request

**Pull request merged**